server/pulp/plugins/rsync/configuration.py

```diff
@@ -128,6 +128,12 @@ def validate_config(repo, config, config_conduit):
     """
     _config = config.flatten()  # now config is {}
     err_list = []
+
+    if "rsync_extra_args" in _config:
+        (valid, err) = TypeValidation([list])(_config["rsync_extra_args"])
```

Comment:
Upon testing, I got a few errors.  This line needs to be

```(valid, err) = TypeValidation([list])(_config["rsync_extra_args"], _config)```

instead.  After this change, it works great.